packages/sprotty-library/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/********************************************************************************
+ * Copyright (c) 2024 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+export * from './flowchart';
```

Comment:
This won't work well when we start adding other diagram types because all the model element interfaces will be mixed up. I propose to wrap this in a namespace (like [we did in Langium](https://github.com/eclipse-langium/langium/blob/e78aebac234b367a7724d711a52477c900ad8d1b/packages/langium/src/utils/index.ts#L17-L22)). Once we migrate to ESM, we can add additional exports for each diagram type.